src/renderer/components/ft-share-button/ft-share-button.js

```diff
@@ -62,6 +62,19 @@ export default defineComponent({
       return this.$store.getters.getCurrentInvidiousInstance
     },
 
+    selectedUserPlaylist: function () {
+      if (this.playlistId == null) { return null }
+      if (this.playlistId === '') { return null }
```

Comment:
These two `if`s can be combined into a single `if`.
```suggestion
      if (this.playlistId == null || this.playlistId === '') { return null }
```